web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -1938,6 +1938,11 @@ private BaseQueryResponse next(final String id, boolean checkForContentLookup) {
         RunningQuery query = null;
         Query contentLookupSettings = null;
         try {
+            if (!id.matches("[a-z\\d-]+")) {
```

Comment:
Extract common pattern to a class level variable. Additionally, do we need 'a-z' or just 'a-f' since we're expecting uuids?

Reply:
a-f seems fine to me, I'll make that adjustment.